### ISSUE TITLE
New Outlook for Mac, Outlook Legacy for Mac: Outlook loses keyboard focus when add-in pane is pinned in Compose mode

### ISSUE THREAD

**Issue by jacifkusto** [OfficeDev/office-js#6378]
We are developing an Outlook add-in for Compose mode. Recently we've started to receive complaints that Outlook loses keyboard focus when our pane is pinned. See the steps leading to the issue below. Please note that all Outlook add-ins are affected.

## Your Environment
* Platform: Mac
* Host: New Outlook for Mac, Outlook Legecy for Mac
* Office version number: 16.103.4 (25120717)
* Operating System: macOS
 
## Expected behavior
Outlook should not lose keyboard focus when a pinned add-in pane is open.

## Current behavior
Outlook loses keyboard focus.

## Steps to reproduce
1. In Outlook Online or New Outlook, create a new mail.
2. Start any Outlook add-in that works in Compose mode, for example Script Lab for Outlook.
3. Pin the add-in pane.
4. Close the email that you created in step 1.
5. Create a new mail again. As you see, the keyboard focus is not in the To field, where it should be.
6. Close the email.
7. Click New or select any email in your Inbox and click Reply or Forward. As you see, the keyboard focus is neither in the Body field (in case of Reply) nor in the To field (in case of New or Forward), where it is supposed to be.

## Context
As you understand, this current behavior affects many users, we are receiving complaints.
